tests/5.1/error/test_error_message.c

```diff
@@ -17,12 +17,11 @@
 int errors, i;
 
 int test_error_message() {
-    char msg[] = "error message"
     i = 0;
-    OMPVV_INFOMSG("If successful, should print "error message": ");
-    #pragma omp parallel
+    OMPVV_INFOMSG("If successful, test should print an \"error message success\" at the beginning of the test");
+    #pragma omp single
```

Comment:
@nolanbaker31 
Use omp single inside of omp parallel (not instead of). We want to create a team of threads and then have just 1 thread modify the value.